test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow.sh

```diff
@@ -4,48 +4,59 @@
 
 set -ex
 
-# Function to install kfctl cli
-install_kfctl(){
+# Function to install kustomize
+install_kustomize(){
+    KUSTOMIZE_VERSION="v4.5.7"
+    KUSTOMIZE_URL="https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.5.7/kustomize_v4.5.7_linux_amd64.tar.gz"
 
-    KFCTL_VERSION="v1.0.2"
-
-    if ! command -v kfctl &> /dev/null
+    if ~ command -v kustomize &> /dev/null
     then
-        KFCTL_URL=https://github.com/kubeflow/kfctl/releases/download/${KFCTL_VERSION}/kfctl_${KFCTL_VERSION}-0-ga476281_linux.tar.gz
-        curl --silent --location ${KFCTL_URL} -o /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz
-        tar -xvf /tmp/kfctl_${KFCTL_VERSION}_linux.tar.gz -C /tmp --strip-components=1
-        if ! [ -x "$(command -v sudo)" ]; then
-            mv /tmp/kfctl /usr/local/bin
+        wget -O /tmp/kustomize_${KUSTOMIZE_VERSION}_linux.tar.gz ${KUSTOMIZE_URL}
+        tar -xvf /tmp/kustomize_${KUSTOMIZE_VERSION}_linux.tar.gz -C /tmp/
+        if ~ [ -x "$command -v sudo" ]; then
+            mv /tmp/kustomize /usr/local/bin
         else
-            sudo mv /tmp/kfctl /usr/local/bin
+            sudo mv /tmp/kustomize /usr/local/bin
         fi
     fi
 }
 
-# Function to install kubeflow in EKS cluster 
+# Function to install kubeflow in EKS cluster using kustomize
 setup_kubeflow(){
+    TIMEOUT=600s
 
-    local REGION=$1
-    KUBEFLOW_URL="https://raw.githubusercontent.com/aws/deep-learning-containers/master/test/dlc_tests/eks/eks_manifest_templates/kubeflow/kfctl_aws.yaml"
-    CONFIG_FILE=kfctl_aws.yaml
-    wget -O ${CONFIG_FILE} ${KUBEFLOW_URL} 
-    sed -i -e 's/<REGION>/'"${REGION}"'/' ${CONFIG_FILE}
-    kfctl apply -V -f ${CONFIG_FILE}
-}
+    # clones manifests from kubeflow github into a folder named manifests
+    git clone https://github.com/kubeflow/manifests.git
 
-# Function to install mpi operator in EKS cluster
-install_mpi_operator() {
+    echo "> Installing cert-manager"
```

Comment:
is this component required?

Reply:
I think we should include it. It was included in the initial kfctl_aws.yaml file.